packages/scripts/config/baseModuleRollupConfig.js

```diff
@@ -0,0 +1,29 @@
+const baseRollupConfig = require('./baseRollupConfig');
```

Comment:
Style thought (and it doesn't _really matter_ because these files aren't accessed directly or often):

What if we named these three files:

- `rollup.config.base.js`
- `rollup.config.plugin.js`
- `rollup.config.module.js`

It reads a bit better to me.  I was reading where these files were being imported and the specific type (base/plugin/module) wasn't standing out to me in the `require()` statement.

or (I don't prefer this, I like `rollup.config` prefixes)

- `base.rollup.config.js`
- `plugin.rollup.config.js`
- `module.rollup.config.js`

or


- `rollup.config.base.js`
- `rollup.config.base.plugin.js`
- `rollup.config.base.module.js`

thoughts?

Reply:
I do like the style of `rollup.config.base.js`. Makes it easy to parse that it is a rollup config file. I'm going to do it in a separate PR to make things a little more manageable.

Reply:
I decided to do it in this PR instead.